libs/payments/cart/src/lib/checkout.service.ts

```diff
@@ -227,13 +227,16 @@ export class CheckoutService {
     await this.customerChanged(uid);
 
     if (cart.couponCode) {
-      const subscriptionMetadata = {
-        ...subscription.metadata,
-        [STRIPE_CUSTOMER_METADATA.SubscriptionPromotionCode]: cart.couponCode,
-      };
-      await this.subscriptionManager.update(subscription.id, {
-        metadata: subscriptionMetadata,
-      });
+      await this.subscriptionManager.update(
+        subscription.id,
+        {
+          metadata: {
+            [STRIPE_CUSTOMER_METADATA.SubscriptionPromotionCode]:
```

Comment:
This says `STRIPE_CUSTOMER_METADATA` but it's updating a subscription. Is that correct?

Reply:
Ah, I'll move `SubscriptionPromotionCode` into `STRIPE_SUBSCRIPTION_METADATA`.